docker/chainnode/entrypoint.sh

```diff
@@ -99,13 +103,15 @@ if [[ ! -z $PRIVATE_KEY ]]; then
     if [[ $exitCode -ne 0 ]]; then
       echo "An account with different address already exists in $KEYSTORE_DIR"
       echo "Please consider remove account in keystore" \
-        "or unset PRIVATE_KEY environment variable"
+        "or unset the private key environment variable"
       exit 1
     fi
     set -e
   fi
   rm ./private_key
   unset PRIVATE_KEY
+elif [[ "$mine" = "true" ]]; then
```

Comment:
```suggestion
# Bypassing unset private key environment 
```
I think we can add a comment for explaining

Reply:
As the log is descriptive below, I don't think it is necessary to add comment here